### DISCUSSION TITLE
Proxy seem to be working but no json generated

### DISCUSSION THREAD

**Discussion by Riukotashi** [Xzandro/sw-exporter#466] [Q&A]
I'm trying to make this exporter working.
I've tried to run it on windows or on linux but it never work.
Am I missing something 
I have seen an recent video showing that it works without https now but it doesn't seem to work on me.
Everything looks fine i can start and connect to the game but no json is generated.
I have laucnh tcpdump to see if there were any packets and I saw a lot of packet on my computer but it did not generate any files.

I'm using the version 0.0.52

Do you have any ideas ?

**Comment by NoSleepCrew** [comment]
I am having the same issue. I have tried nearly every combo of settings and such. New certs, https on and off, new certs for https, changed port from 8080 and then back.  I tried the "work around" before the latest update to 0.0.52.
I can start the proxy and I can start the game, but I never get the json. I can play the game w/o issue and if I stop the proxy the game stops responding. Then restart the proxy and the game responds again but i never get a json.
Any and all help is appreciated.
